src/Strengths/Strengths.js

```diff
@@ -4,25 +4,28 @@ import Rankings from './Rankings'
 import Legend from '../Charting/Legend'
 import Legends from '../Core/LegendTexts'
 
-const Strengths = ({match}) => {
+const Strengths = ({ match }) => {
     const LegendText = Legends.Strengths
+    const legende = <Legend
+        title={LegendText.Explanation.title}>
+        {LegendText.Explanation.text.map((text, index) =>
+            <p style={{ fontWeight: index === 1 && 'bold' }} key={text}>{text}</p>)}
+    </Legend>
```

Comment:
Die Formattierung (Einrückungen) sieht hier ein bisschen komisch aus.  - hast du die Datei mit autoformat eingerückt?

Reply:
Nein Manual, das Autoformat macht keinen Bruch in einer Zeile.
Ich habe es so gemacht, um übersichtlich zu werden.
Ich habe es wieder auf eine Zeile umgewandelt